packages/cosmo-dycore/package.py

```diff
@@ -45,6 +45,7 @@ class CosmoDycore(CMakePackage):
     version('dev-build', branch='master')
     version('mch', git='git@github.com:MeteoSwiss-APN/cosmo.git', branch='mch')
     version('gt2', git='git@github.com:havogt/cosmo.git', branch='gt2')
+    version('test', git='git@github.com:elsagermann/cosmo.git', branch='add_spack_tests')
```

Comment:
whenever possible I would remove this

Reply:
yes of course, it is just for testing with Jenkins, then I'll remove them before merging!